src/main/java/teletubbies/model/UserPrefs.java

```diff
@@ -56,8 +61,18 @@ public UserProfile getUserProfile() {
         return userProfile;
     }
 
-    public void setUserProfile(UserProfile userProfile) {
+    @Override
+    public boolean isProfileSet() {
+        return this.isProfileSet;
+    }
+
+    public void setUserProfile(UserProfile userProfile) throws UserRoleSetException {
         requireNonNull(userProfile);
+        System.out.println(isProfileSet);
```

Comment:
Perhaps remove or log instead

Reply:
haha slipped through again, thanks for the spot

Reply:
```suggestion
```